rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/cmd/sched/SubmitJobCommand.java

```diff
@@ -83,9 +88,60 @@ public void execute(ApplicationContext currentContext) throws CLIException {
                         e,
                         currentContext);
         }
+
+    }
+
+    private void validateFilePath(ApplicationContext currentContext) {
+        if (!checkFilePathValid(pathname)) {
+            throw new CLIException(REASON_INVALID_ARGUMENTS, String.format("'%s' is not a valid file.", pathname));
+        }
+
+        if (!checkFileExists(pathname)) {
+            throw new CLIException(REASON_INVALID_ARGUMENTS, String.format("'%s' does not exist.", pathname));
+        }
+
+        if (!checkFileNotEmpty(pathname)) {
+            writeLine(currentContext, "File " + pathname + " is empty ");
+            throw new CLIException(REASON_FILE_EMPTY, String.format("'%s' is empty.", pathname));
+        }
+
     }
 
     private Map<String, String> map(String variables) {
         return jobKeyValueTransformer.transformVariablesToMap(variables);
     }
+
+    private Boolean checkFileNotEmpty(String pathname) {
```

Comment:
can you rename this to isFileNotEmpty , the check doesn't indicate when it returns true vs false